KarhooUISDK/Classes/Payment/AddCard/AdyenCardRegistrationFlow.swift

```diff
@@ -120,18 +120,19 @@ extension AdyenCardRegistrationFlow: DropInComponentDelegate {
                 finish(result: .completed(value: .didFailWithError(nil)))
                 return
             }
-            submitPayments(dropInJson: adyenData)
+            submitPayments(dropInJson: adyenData, storePaymentMethod: data.storePaymentMethod)
         } catch let error {
             adyenDropIn?.stopLoading()
             didFail(with: error, from: component)
         }
     }
 
-    private func submitPayments(dropInJson: [String: Any]) {
+    private func submitPayments(dropInJson: [String: Any], storePaymentMethod: Bool) {
         var adyenPayload = AdyenDropInPayload()
         adyenPayload.paymentMethod = dropInJson
         adyenPayload.amount = adyenAmout
         adyenPayload.additionalData = ["allow3DS2": "true"]
+        adyenPayload.storePaymentMethod = storePaymentMethod
```

Comment:
Does this need a test to check that the right value is set on the request?

Reply:
@jsm20 there's some tech debt that Im addressing today to add unit tests to this extension! Should be up in a PR tomorrow but didn't want to block Nur in the meantime :)